src/helpers/render-document-file.js

```diff
@@ -75,8 +224,42 @@ export const buildImage = async (docxDocumentInstance, vNode, maximumWidth = nul
         internalRelationship
       );
 
-      const imageBuffer = Buffer.from(response.fileContent, 'base64');
-      const imageProperties = sizeOf(imageBuffer);
+      // Add validation before calling sizeOf
+      if (!imageBuffer || imageBuffer.length === 0) {
+        // eslint-disable-next-line no-console
+        console.error(`[ERROR] buildImage: Empty image buffer for ${vNode.properties.src}`);
+        return null;
+      }
+
+      // Check if we got HTML instead of image data (common with Wikimedia errors)
+      const firstBytes = imageBuffer.slice(0, 20).toString('utf8');
+      if (firstBytes.startsWith('<!DOCTYPE') || firstBytes.startsWith('<html')) {
+        // eslint-disable-next-line no-console
+        console.error(
+          `[ERROR] buildImage: Received HTML instead of image data for ${vNode.properties.src}`
+        );
+        return null;
+      }
+
+      let imageProperties;
+      try {
+        imageProperties = sizeOf(imageBuffer);
+        if (!imageProperties || !imageProperties.width || !imageProperties.height) {
+          // eslint-disable-next-line no-console
+          console.error(
+            `[ERROR] buildImage: Invalid image properties for ${vNode.properties.src}:`,
+            imageProperties
+          );
+          return null;
```

Comment:
It's good we are checking, but what is the desired output we get from image-size. If we are sure to return height and width properties, the check might not be necessary.